### PR TITLE
Add: property-read/write to class property list

### PR DESCRIPTION
**Pull request by halftan**

### CODE REVIEW

**Comment by mkusher** [comment]
cool, can you add some simple feature test like this https://github.com/mkusher/padawan.php/blob/master/features/class-constructor.feature ?

**Comment by halftan** [comment]
sure, I'll update this PR once it's done.

**Comment by halftan** [comment]
add test e2baa23

File: features/class-constructor.feature [hunk 1]
```diff
@@ -24,3 +24,39 @@ Feature: Processing constructor assignments
         Then I should get:
             | Name | Signature |
             | prop | SomeClass |
+
+    Scenario: Get properties list from classdoc
```
Comment (mkusher) [inline features/class-constructor.feature]: I think it's better to move this scenario to file `class-phpdoc.feature`.

File: features/class-constructor.feature [hunk 2]
```diff
@@ -24,3 +24,39 @@ Feature: Processing constructor assignments
...
+        """
+        <?php
+
+        /**
+         * @property SomeClass $stdProp
+         * @property-read string $roProp
+         * @property-write string $woProp
+         * @property bool $boolProp
+         * @property int $intProp
+         * @property int|bool|null $mixedProp
```
Comment (mkusher) [inline features/class-constructor.feature]: it's more like a bug, than feature :D

**Comment by halftan** [comment]
updated e367829

**Pull request merged**